chimedb/dataflag/orm.py

```diff
@@ -261,6 +261,12 @@ class DataFlagOpinion(DataSubset):
     client = pw.ForeignKeyField(DataFlagClient)
     revision = pw.ForeignKeyField(DataRevision, backref="opinions")
 
+    class Meta:
+        indexes = (
+            # create a unique index
+            (("type", "user", "start_time", "finish_time"), True),
```

Comment:
I think you should also index on `revision` in here.

Reply:
yes